api/src/main/java/org/apache/gravitino/MetadataObjects.java

```diff
@@ -159,7 +160,7 @@ public static MetadataObject parse(String fullName, MetadataObject.Type type) {
    * @param names The names of the metadata object
    * @return The parent full name if it exists, otherwise null
    */
-  public static String getParentFullName(List<String> names) {
+  private static String getParentFullName(List<String> names) {
```

Comment:
It is better not to change the scope from public to private, this will break the compatibility.

Reply:
This is my rollback change in the https://github.com/apache/gravitino/pull/5113